pkg/client/images.go

```diff
@@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
 	err = c.r.Delete().Resource("images").Name(name).Do().Error()
 	return
 }
+
+// UpdateSignatures allows to modify signatures of given image.
+func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
```

Comment:
Yeah I think the wrapper that makes sense and models what we'd do for end
users.  Ref to image, although we may want to make the signature name be
the unique verification (I don't know that I want a special name for a
signature).

On Mon, Jun 13, 2016 at 1:25 PM, David Eads <notifications@github.com>
wrote:

> In pkg/client/images.go
> <https://github.com/openshift/origin/pull/9181#discussion_r66832256>:
>
> > @@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
> >  	err = c.r.Delete().Resource("images").Name(name).Do().Error()
> >  	return
> >  }
> > +
> > +// UpdateSignatures allows to modify signatures of given image.
> > +func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
>
> Seems like this is more CreateSignatures. It's updating an underlying
> object, but you're actually creating a thing, right?
>
> @smarterclayton <https://github.com/smarterclayton> I think I'd do a
> wrapper on ImageSignature to register it as an API type that can include
> the image it wants to use. Similar to api.Binding used for pods/binding.
> Are you attached to stripped down Image instead? I think its less obvious.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9181/files/9ff4bf43548c758b6767b639b335681285fece48#r66832256>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p2jy59h6ZZUdS7qocfm2QYiYn8vHks5qLZKSgaJpZM4IvEFx>
> .
>